addons/website_event_sale/tests/test_frontend_buy_tickets.py

```diff
@@ -125,3 +154,66 @@ def test_buy_last_ticket(self):
         self.start_tour("/", 'event_buy_last_ticket')
 
     # TO DO - add public test with new address when convert to web.tour format.
+
+
+class TestUnitPriceUi(TestUiCommon):
+    """Test of the controller used to display the unit price.
+
+    This price depends on the selected quantity and the user groups. The last determines whether the price should be
+    tax included or excluded. """
+    def setUp(self):
+        super().setUp()
+        self.headers_json = {"Content-Type": "application/json", }
+        self.company_currency = self.env.company.currency_id
+        self.precision_delta = self.company_currency.rounding
+
+    def setup_display_tax_incl(self):
+        self.env['res.config.settings'].create({
+            'show_line_subtotals_tax_selection': 'tax_included',
+            'group_show_line_subtotals_tax_excluded': False,
+            'group_show_line_subtotals_tax_included': True,
```

Comment:
referring to ":thinking:", do you need both lines, then?

Reply:
Yes, following the comment res_config_settings in account model (see previous answer), I have also simplified to only set the tax included flags.

Reply:
Actually, when removing "'group_show_line_subtotals_tax_excluded': False,", there is an error on the runbot (but not locally):
ValidationError(_("A user cannot have both Tax B2B and Tax B2C.\n"
odoo.exceptions.ValidationError: A user cannot have both Tax B2B and Tax B2C.
You should go in General Settings, and choose to display Product Prices
either in 'Tax-Included' or in 'Tax-Excluded' mode
(or switch twice the mode if you are already in the desired one).

I suppose the last remark to switch twice the mode is what it does when the 2 lines are presents. So I have re-added the line (which is also present in another test). What do you think ?

Reply:
OK, let's keep it here then, but it's not necessary when testing for the current values, right?

Reply:
Yes it shouldn't be necessary as the information is redundant. The test is currently simplified and the test passes.